### ISSUE TITLE
Implementation/suggestion

### ISSUE THREAD

**Issue by CZamberti** [chanthamonsta/vr_wheelchair_simulator#1]
hello I'm working on a project that involves the use of a VR wheelchair and I'm trying to implement (unsuccessfully) the backward movement in the OVRGrabber script, starting from the original and looking at yours to understand its logic. Any suggestions?

**Comment by chanthamonsta** [comment]
Hi @CZamberti . This was a final project I did with a group of friends for our vr class so sorry if my suggestion isn't helpful. But I think what we did was take whatever direction our wheel moved and made some arbitrary movement in that direction. If you look at OVRGrabber.cs and checkout moveForward and MoveGrabbedObject functions that may be helpful.

**Comment by chanthamonsta** [comment]
Also if you are using our original, is the backwards movement not working for you? Or is that a feature you are trying to add?

**Comment by CZamberti** [comment]
Thanks for answering. I'm just trying to implement it. I was working on what you said, trying to invert the direction of the player, working on the forward movement (now I don't have the code opened so I can't tell you the lines I was working on). 
With your original OVRGrabber, using it directly in your project (imported and upgraded to the new unity version), the wheelchair can only be moved forward if controlled with the grabber.

⁣Ottieni BlueMail per Android ​

Il giorno 3 giu 2020, 08:14, alle ore 08:14, chanthamoney <notifications@github.com> ha scritto:

**Comment by chanthamonsta** [comment]
I remember talking to our Professor about this and what was hard the point we grab the wheel and what direction the rotation spun. If you can figure that out and then use that rotation to figure out if it's forward and backward I think that will be able to work!

**Comment by CZamberti** [comment]
Thanks a lot. Unluckily, I'm still not able to figure out where the problem is and where I have to grab the wheel to switch the movement direction. Meanwhile, I'm working on another part of the project. When I'll work on the player again, I'll let you know.
Great project, btw

⁣Ottieni BlueMail per Android ​

Il giorno 4 giu 2020, 23:51, alle ore 23:51, chanthamoney <notifications@github.com> ha scritto:

**Comment by chanthamonsta** [comment]
Yeah it's hard to figure out the rotation because you gotta think about world space and then your player controller space to make sure it's in the same coordinat plane! If you figure it out though let me know! I would love to see a solution! :)

**Comment by CZamberti** [comment]
I've been working in VR only for two months, so you can understand how difficult it can be for me. 
I also noticed your sensors were positioned differently from mines while working at this project, because the player appears to be bigger than the world space and the camera rig is too far from the wheels: these settings depend on the user physical configuration of the hardware, I guess

⁣Ottieni BlueMail per Android ​

Il giorno 5 giu 2020, 00:16, alle ore 00:16, chanthamoney <notifications@github.com> ha scritto:

**Comment by chanthamonsta** [comment]
Yeah It was a weird configuration set up that we did. We wanted to make sure that the person felt like they were on a chair and lower than the space around them

**Comment by CZamberti** [comment]
With my config, with the sensors higher than my chair, I can see the world from the high. Tomorrow I will try putting my sensors in another place. 
My project too has the same goal: make people feel what is the world seen from a disabled person's eyes, but mine consists of a house with some special equipment and a domotic-like system, with smartphone/voice commands integrations. It was almost done, but now I have to make it vr ready and the wheelchair part I think is the hardest to implement

⁣Ottieni BlueMail per Android ​

Il giorno 5 giu 2020, 00:29, alle ore 00:29, chanthamoney <notifications@github.com> ha scritto:

**Comment by chanthamonsta** [comment]
Good luck with the wheel chair part! When the project is finished I would love to share it with my group and show it to them :) 
Link us your repo too ! 💯

**Comment by CZamberti** [comment]
I don't have a repo right now, because the code can't be available yet - I'm doing a internship and I don't have reached my goals yet. It's very messy too. When I will be done, I'll share it if possible :) 

⁣Ottieni BlueMail per Android ​

Il giorno 5 giu 2020, 01:06, alle ore 01:06, chanthamoney <notifications@github.com> ha scritto:
